tajo-storage/tajo-storage-pgsql/pom.xml

```diff
@@ -224,10 +224,11 @@
   <profiles>
     <profile>
       <!-- testing-postgresql-server only supports x86-64 -->
-      <id>64bit-tests</id>
+      <id>64bit-intel-tests</id>
       <activation>
         <os>
-         <arch>x86_64</arch>
+          <name>x86_64</name>
```

Comment:
As far as I know, ``name`` is intended to describe operation system name like ``linux`` or ``mac os x``.